### PR TITLE
(docs) Clean up CDN list

### PR DESCRIPTION
**Pull request by Hirse**
### Changes
- Update cdn links to latest version
- Add extra-language example for all CDNs

### Checklist
- [x] ~Added markup tests, or they don't apply here because...~
- [x] ~Updated the changelog at `CHANGES.md`~
- [x] ~Added myself to `AUTHORS.txt`, under Contributors~

### CODE REVIEW

**Comment by joshgoebel** [comment]
> Add extra-language example for all CDNs

You feel this needs to be duplicated 3 times in close proximity?

**Comment by Hirse** [comment]
> > Add extra-language example for all CDNs
> 
> You feel this needs to be duplicated 3 times in close proximity?

When I see it only for one of the CDNs, it looks like only that one supports adding extra languages, which is not true.

File: README.md [hunk 1]
```diff
@@ -243,31 +243,32 @@ A prebuilt version of highlight.js bundled with many common languages is hosted
 **cdnjs** ([link](https://cdnjs.com/libraries/highlight.js))
 
 ```html
-<link rel="stylesheet"
-      href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.4.0/styles/default.min.css">
-<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.4.0/highlight.min.js"></script>
+<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.4.1/styles/default.min.css">
+<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.4.1/highlight.min.js"></script>
```
Comment (joshgoebel) [inline README.md:247]: Is there a reason you added https to JSDevlir but not here?

Comment (Hirse) [reply README.md]: No. Added to all of them now.

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
Thanks.

**Pull request merged**